### PR TITLE
Reset test database

### PR DESCRIPTION
**Pull request by nbuonin**

### CODE REVIEW

File: locustempus/settings_travis.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
     DATABASES = {
         'default': {
             'ENGINE': 'django.contrib.gis.db.backends.postgis',
-            'NAME': 'test_locustempus',
+            'NAME': 'locustempus',
```
Comment (sdreher) [inline locustempus/settings_travis.py:17]: why did this change?

Comment (sdreher) [reply locustempus/settings_travis.py:17]: Oh...was it creating test_test_locustempus?

Comment (nbuonin) [reply locustempus/settings_travis.py:17]: yeah, exactly

Comment (sdreher) [reply locustempus/settings_travis.py:17]: Looks great - merging!

**Pull request merged**